controllers/construct/mongodbstatefulset.go

```diff
@@ -245,7 +245,7 @@ func DefaultReadiness() probes.Modification {
 	return probes.Apply(
 		probes.WithExecCommand([]string{readinessProbePath}),
 		probes.WithFailureThreshold(40),
-		probes.WithInitialDelaySeconds(5),
+		probes.WithInitialDelaySeconds(15),
```

Comment:
note: if that helps, this is not the solution, just a measure to figure out the cause